tools/report_ci_failure.py

```diff
@@ -46,6 +49,8 @@ def report(self, branch, commit, infourl=None, job_name=None):
                 build logs.
             job_name (str): name of the failed ci job.
         """
+        if branch != 'master' and not self._stable_regex.search(branch):
```

Comment:
why not just `branch.startswith('stable/')?

I hate regular expressions so I avoid them every time I can :)

Reply:
re removed in 160e13fce